lib/iso_spinner.py

```diff
@@ -114,13 +114,13 @@ def _create_spin_kickstart(self):
                 f.write(repo)
 
             f.write("%packages\n")
-            package_group_list = self.config.get('package_group_list')
-            for group in self.config.get('hostos_packages_groups').keys():
-                group = "@%s" % group
-                if group not in package_group_list:
-                    package_group_list.append(group)
-            for package_group in package_group_list:
-                f.write(package_group + "\n")
+            groups = self.config.get('automated_install_package_groups')
+            for hostos_group in self.config.get('hostos_packages_groups').keys():
+                hostos_group = "@%s" % hostos_group
+                if hostos_group not in groups:
+                    groups.append(hostos_group)
+            for group in groups:
+                f.write(group + "\n")
```

Comment:
Regarding commit message:
"Also, implemented a related small variable renaming in package group creation code
in ISO creator."
I'd write something simpler:
"Also, renamed related variables in ISO creator."